### ISSUE TITLE
Add date and time of last save.

### ISSUE THREAD

**Issue by PritchardBarry** [andymac-2/trakMap#25]
Can we add the date and time of last save at the top left, under the controls. 

Helps when working with files across areas.

**Comment by andymac-2** [comment]
Is this like milestone map? Do we need a vertical line for the current day?

**Comment by PritchardBarry** [comment]
No. This is because I sometimes moves the files from device to device and got confused on which file was the last updated.

**Comment by andymac-2** [comment]
Can this problem be solved by sorting your files by date modified on the
machine? Either that or storing the data in Google drive or OneDrive so
that all devices can access it?

I ask because this is a particularly strange feature to request that
appears to be tangential to the core functionality. This problem in usually
solved by storing files in a central place, or using a version control
system. For non technical users, Dropbox would be ideal, and it keeps old
versions of a file for 30 days. For intermediate users: tortoiseSVN, and
for technical users, git.

**Comment by PritchardBarry** [comment]
Agreed but it is also useful for when we print the output so we know the currency of the output.
